EKF/estimator_interface.cpp

```diff
@@ -78,6 +80,13 @@ EstimatorInterface::EstimatorInterface():
 	_delta_ang_prev.setZero();
 	_delta_vel_prev.setZero();
 	memset(_vibe_metrics, 0, sizeof(_vibe_metrics));
+	memset(&_mag_sample_delayed, 0, sizeof(_mag_sample_delayed));
```

Comment:
@priseborough I'm never sure if this works because the class embodies vector class instances.